modules/ai/preferences.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Elementor\Modules\Ai;
+
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Preferences {
+
+	const ENABLE_AI = 'elementor_enable_ai';
+
+	/**
+	 * Register actions and hooks.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_action( 'profile_personal_options', function ( \WP_User $user ) {
```

Comment:
Please replace with `personal_options` to show on every user option page

```suggestion
		add_action( 'personal_options', function ( \WP_User $user ) {
```